test/meson.build

```diff
@@ -77,13 +77,15 @@ test(
         '-V'
     ]
 )
-test(
-    'check version info in zckdl',
-    zckdl,
-    args: [
-        '-V'
-    ]
-)
+if curl_dep.found()
+    test(
+        'check version info in zckdl',
```

Comment:
If curl_dep is not found, and is thus a disabler, then based on the above suggestion, zckdl links to the disabler and then *becomes* a disabler too.

So, this `test()` is trying to run a disabler, and hence the test is a disabler too, i.e. there is no `'check version info in zckdl'` test. So again, you do not need the modifications to this file. disablers handle it all for you.